### PR TITLE
Remove -release option from the Scala 3 PC

### PR DESCRIPTION
**Pull request by tgodzik**
Previously, in a number of scenarios this option would break the presentation compiler for Scala 3. Now we filter it out since it's not important for the frontend phases.

### CODE REVIEW

File: mtags/src/main/scala-3/scala/meta/internal/pc/ScalaPresentationCompiler.scala [hunk 1]
```diff
@@ -58,10 +60,19 @@ case class ScalaPresentationCompiler(
       using ec
     )
 
+  private def removeDoubleOptions(options: List[String]): List[String] =
+    options match
+      case head :: _ :: tail if forbiddenDoubleOptions(head) =>
+        removeDoubleOptions(tail)
+      case head :: tail => head :: removeDoubleOptions(tail)
+      case Nil => options
```
Comment (tanishiking) [inline mtags/src/main/scala-3/scala/meta/internal/pc/ScalaPresentationCompiler.scala:68]: Why not

```suggestion
      options.filterNot(forbiddenDoubleOptions)
```

?

Comment (tanishiking) [reply mtags/src/main/scala-3/scala/meta/internal/pc/ScalaPresentationCompiler.scala:68]: Ahh, nevermind, `removeDoubleOptions` transforms `List("-release", "8", ...)` to `List(...)`

**Review by tanishiking — APPROVED** [review]
LGTM

```
-release              Compile code with classes specific to the given version
                      of the Java platform available on the classpath and emit
                      bytecode for this version.
```

Indeed it's not important for `frontend` phases :+1:

**Review by kpodsiad — APPROVED** [review]
Great!

File: mtags/src/main/scala-3/scala/meta/internal/pc/ScalaPresentationCompiler.scala (hunk 1 above)
Comment (tgodzik) [reply mtags/src/main/scala-3/scala/meta/internal/pc/ScalaPresentationCompiler.scala:68]: I had to get a bit more creative here :sweat_smile:

**Pull request merged**